spec/controlbox.js

```diff
@@ -1087,6 +1087,33 @@
             // XXX: Awaiting more tests, close it again for now...
             panel.$el.find('a.toggle-xmpp-contact-form').click();
         }));
+
+        it("can be added to roster and check for case-sensivity", mock.initConverse(function (_converse) {
```

Comment:
The `describe` section and the `it` section should together read like a sentence.

For example: "The 'Add Contact' widget opens up an add form when you click on it.

What you have done is to create the following sentence:
"The 'Add Contact widget" can be added to roster and check for case-sensitivity"
which doesn't make sense.